lib/utils/clock.js

```diff
@@ -31,11 +32,26 @@ videoTsToAudioTs = function(timestamp, sampleRate) {
   return secondsToAudioTs(videoTsToSeconds(timestamp), sampleRate);
 };
 
+/**
+ * Adjust ID3 tag or caption timing information by the timeline pts values
+ * (if keepOriginalTimestamps is false) and convert to seconds
+ */
+metadataTsToSeconds = function(timestamp, timelineStartPts, keepOriginalTimestamps) {
+  var newTimestamp = timestamp;
```

Comment:
Can reuse `timestamp`

Reply:
The function can actually be one line: `return videoTsToSeconds(keepOriginalTimestamps ? timestamp : timestamp - timelineStartPts);`